pkg/ctl/get/nodegroup.go

```diff
@@ -62,6 +54,20 @@ func doGetNodeGroup(cmd *cmdutils.Cmd, ng *api.NodeGroup, params *getCmdParams)
 		ng.Name = cmd.NameArg
 	}
 
+	if ng.Name == "" {
+		// avoid validation errors when listing all nodegroups
+		cfg.NodeGroups = nil
```

Comment:
While this skips the validation for node groups, it'd be ideal if we could avoid calling `ValidateClusterConfig` for this usage of the command as there's no node group input to validate here.

Reply:
At the moment the idea is that `ValidateClusterConfig` is mean to be call in all instances, whether `ClusterConfig` instance was constructed by the command, or parsed from a config file. I would rather review how it's constructed. I'll see if there is an easy fix to this that is a more coherent.